modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -165,7 +165,7 @@ def buildProject(Map options = [:]) {
     lock("$repoName-$NODE_NAME-test") {
       if (hasDatabase()) {
         stage("Set up the database") {
-            runRakeTask("db:drop db:create db:schema:load")
+            runRakeTask("db:drop db:create db:schema:load_if_ruby db:structure:load_if_sql")
```

Comment:
Rails supplies `db:setup` which does these steps – https://github.com/rails/rails/blob/master/activerecord/lib/active_record/railties/databases.rake#L178-L179

Furthermore, it supplies `db:reset` which looks to do everything we need (though I haven't tested) – https://github.com/rails/rails/blob/master/activerecord/lib/active_record/railties/databases.rake#L143-L144